Dockerfile.assisted-test-infra

```diff
@@ -1,4 +1,15 @@
-FROM quay.io/edge-infrastructure/assisted-service:latest AS service
+# Generate python client
+FROM quay.io/edge-infrastructure/swagger-codegen-cli:2.4.18 AS swagger_py
+
+COPY assisted-service/swagger.yaml /
+COPY assisted-service/tools/generate_python_client.sh .
+RUN chmod +x ./generate_python_client.sh && SWAGGER_FILE=swagger.yaml OUTPUT=/build ./generate_python_client.sh
+
+FROM registry.access.redhat.com/ubi9/python-311:latest AS pybuilder
+COPY --chown=1001:0 ./assisted-service /assisted-service
+COPY --from=swagger_py --chown=1001:0 /build /assisted-service/build
+RUN python3 -m pip install -I pip && pip3 install -I -r /assisted-service/dev-requirements.txt
+RUN cd /assisted-service/build && python3 ../tools/client_package_initializer.py ./ https://github.com/openshift/assisted-service
```

Comment:
nit: In general I think we should avoid consecutive RUN statements: we can squash them into one and reduce layers